ddtrace/appsec/_handlers.py

```diff
@@ -376,13 +374,18 @@ def _custom_protobuf_getattribute(self, name):
             source_origin=OriginType.GRPC_BODY,
         )
     elif isinstance(ret, MutableMapping):
-        if isinstance(ret, MessageMapContainer) and len(ret):
-            # Patch the message-values class
-            first_key = next(iter(ret))
-            value_type = type(ret[first_key])
-            _patch_protobuf_class(value_type)
-        else:
-            ret = taint_structure(ret, OriginType.GRPC_BODY, OriginType.GRPC_BODY)
+        try:
+            from google._upb._message import MessageMapContainer
```

Comment:
can we move this to the module level?

trying this import + try/except on every getattribute is going to be slow.


```python
MessageMapContainer = None
try:
    from google._upb._message import MessageMapContainer
except ImportError:
    pass


...

def _custom_protobuf_getattribute(self, name):
    ...
    if MessageMapContainer is None:
        return ret

    ...
```


wdyt?